src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
```

Comment:
it's a good practice to use object destructing. Apply for other components as well